uint8 ID
uint64 timestamp
float64[3] compass
float64[3] gyro
float64[3] accel
float64[3] fusionPose
float64[4] fusionQPose
